File: CodeForces/Educational_Round_67/problem_A.cpp
//http://codeforces.com/contest/1187/problem/A
//Accepted

#include <bits/stdc++.h>
using namespace std;

int main()
{

#ifdef DEBUG
    freopen("input.txt", "r", stdin);
    freopen("output.txt", "w", stdout);
#endif
    ios_base ::sync_with_stdio(false);
    cin.tie(0);

    int tc;

    cin >> tc;

    while (tc--)
    {
        int n, s, t, res;

        cin >> n >> s >> t;

        res = n - min(s, t) + 1;

        cout << res << "\n";
    }
    return 0;
}

File: CodeForces/Educational_Round_67/problem_A.java
//http://codeforces.com/contest/1187/problem/A
//Accepted

import java.util.Scanner;

public class problem_A {

    public static void main(String args[]) {
        int tc = 0;

        Scanner sc = new Scanner(System.in);

        tc = sc.nextInt();
        int n, s, t;

        while (tc > 0) {

            n = sc.nextInt();
            s = sc.nextInt();
            t = sc.nextInt();

            System.out.println(n - Math.min(s, t) + 1);

            tc--;
        }

    }
}